ush/test/README.md

```diff
@@ -0,0 +1,95 @@
+## Test bufr_query mapping, Python converter script, and ioda configuration YAML in obsForge
+This is a prototype for testing BUFR to IODA conversion and is still evolving.
+
+## Prerequisite
+- Clone and build obsForge  
+
+   ```       
+      git clone --recursive https://github.com/noaa-emc/obsForge
+   
+      cd ./obsForge/sorc/ioda
+   
+      git remote -v
+   
+      git remote set-url origin https://github.com/jcsda-internal/ioda.git
+   
+      git pull
+   
+      git checkout feature/bufr_in_parallel_emily
+
+      cd ../spoc
+
+      git checkout feature/dump_satwind_goes
+
+      cd ../../
+   
+      ./build.sh
+   ```
+
+- Example: obsForge builds on HERA
+  
+   ```
+      obsForge  /scratch1/NCEPDEV/da/Emily.Liu/EMC-obsForge/obsForge
+   ```
+
+## Elements should be in the working directory from SPOC
+- Required input files:
+  
+   - bufr2ioda_satwnd_amv_goes.py
+     
+   - bufr2ioda_satwnd_amv_goes_mapping.yaml
+     
+   - bufr2ioda_bufr_backend_satwnd_amv_goes.yaml
+     
+   - bufr2ioda_script_backend_satwnd_amv_goes.yaml
+     
+   - testinput/2021080100/gdas.t00z.satwnd.tm00.bufr_d (copied from the global dump)
+
+- Processing shell script:
+   - encodeBufr.sh 
+
+## How to run the test shell script
+- Get the help page for usage
+
+```
+      bufrioda.sh -h
```

Comment:
This should be encodeBufr.sh

Reply:
Ha!  Thanks for reminding me.  I will change it now.